src/asmcnc/production/database/calibration_database.py

```diff
@@ -574,10 +573,6 @@ def send_data_through_publisher(self, sn_for_db, stage_id):
 
         sent = send_csv_to_ftp(csv)
 
-        if not sent:
-            log("Failed to send data for stage id: " + str(stage_id))
```

Comment:
Ah, apologies, I only meant the last "return False" wasn't needed - the log is definitely still useful!

Reply:
The function that 'sent' is being returned to already handles the logging :)